src/test/integration/reload-runner/index.js

```diff
@@ -0,0 +1,72 @@
+const WebSocket = require('ws');
```

Comment:
Is this just an example of how to fetch data from a database? Since its part of `/test/integration` it should written as a mocha test imo and executed in CircleCI

Reply:
This is just an example

Reply:
Think we should rethink the folder structure of this repo. Quite confusing of what is a test/example etc.